pytorch/training/docker/2.0/py3/cu118/Dockerfile.gpu

```diff
@@ -533,6 +540,34 @@ RUN pip install --no-cache-dir -U ${SMD_MODEL_PARALLEL_URL}
 # Install SM Distributed DataParallel binary
 RUN SMDATAPARALLEL_PT=1 pip install --no-cache-dir ${SMD_DATA_PARALLEL_URL}
 
+# Install NCCL-tests (needed for Sagemaker provided health check)
+RUN git clone https://github.com/NVIDIA/nccl-tests.git ${HOME}/nccl-tests \
+    && cd ${HOME}/nccl-tests \
+    && git checkout \
+    && make MPI=1 MPI_HOME=/opt/amazon/openmpi/ CUDA_HOME=/opt/conda NCCL_HOME=/opt/nccl/lib \
+    && cp /root/nccl-tests/build/all_gather_perf /usr/bin \
+    && cp /root/nccl-tests/build/all_reduce_perf /usr/bin \
+    && cp /root/nccl-tests/build/alltoall_perf /usr/bin \
+    && cp /root/nccl-tests/build/broadcast_perf /usr/bin \
+    && cp /root/nccl-tests/build/gather_perf /usr/bin \
+    && cp /root/nccl-tests/build/hypercube_perf /usr/bin \
+    && cp /root/nccl-tests/build/reduce_perf /usr/bin \
+    && cp /root/nccl-tests/build/reduce_scatter_perf /usr/bin \
+    && cp /root/nccl-tests/build/scatter_perf /usr/bin \
+    && cp /root/nccl-tests/build/sendrecv_perf /usr/bin
```

Comment:
`${HOME}/nccl-tests` dir need to be removed

Reply:
removed